composer.lock

```diff
@@ -0,0 +1,4921 @@
+{
```

Comment:
No need to commit this, can be gitignored.

Reply:
Added to gitignore.